src/helpers.ts

```diff
@@ -29,15 +29,34 @@ export enum IncomingEventType {
  */
 export function getTypeAndConversation(body: any): { type?: IncomingEventType; conversationId?: string } {
   if (body.event !== undefined) {
-    const eventBody = body as SlackEventMiddlewareArgs<string>['body'];
+    const { event } = body as SlackEventMiddlewareArgs<string>['body'];
+
+    // Find conversationId
+    const conversationId: string | undefined = (() => {
```

Comment:
This code actually got a bit longer because it fixes some actual bugs (not just a straight refactor). Previously, events of types `pin_added`, `pin_removed`, `call_rejected`, `channel_created`, `channel_rename`, `group_rename`, and `im_created` would not have a `conversationId` in the return value because the conditional checks were incomplete.